### ISSUE TITLE
Updated Report Formats

### ISSUE THREAD

**Issue by mdemaree99** [salesagility/SuiteCRM#2109]
#### Issue

Add a date format for reports  
m/d/Y      allows report to show my current Date Format.  Else time shows up in report for date field.
m/d  -- Allows to show birth-date without year
m     -- Allows to search birth month

**Comment by ijdavie** [comment]
Hi @mdemaree99, 
This should be able to update this with dropdown editor. 
I will raise this as a suggestion, if you would like more information relating to this item you can raise questions on the forums 

https://suitecrm.com/forum/index 

Thanks.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Updated Report Formats](https://trello.com/c/VNqOqaep/78-updated-report-formats)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
